drools-wb-screens/drools-wb-scorecard-xls-editor/drools-wb-scorecard-xls-editor-backend/src/main/java/org/drools/workbench/screens/scorecardxls/backend/server/ScoreCardXLSServiceImpl.java

```diff
@@ -125,67 +126,64 @@ public InputStream load( final Path path,
         }
     }
 
+    @Override
     public Path create( final Path resource,
                         final InputStream content,
                         final String sessionId,
                         final String comment ) {
-        final SessionInfo sessionInfo = getSessionInfo( sessionId );
-        log.info( "USER:" + sessionInfo.getIdentity().getIdentifier() + " CREATING asset [" + resource.getFileName() + "]" );
-
-        try {
-            final org.uberfire.java.nio.file.Path nioPath = Paths.convert( resource );
-            ioService.createFile( nioPath );
-            final OutputStream outputStream = ioService.newOutputStream( nioPath,
-                                                                         commentedOptionFactory.makeCommentedOption( comment,
-                                                                                                                     sessionInfo.getIdentity(),
-                                                                                                                     sessionInfo ) );
-            IOUtils.copy( content,
-                          outputStream );
-            outputStream.flush();
-            outputStream.close();
-
-            return resource;
-
-        } catch ( Exception e ) {
-            throw ExceptionUtilities.handleException( e );
-
-        } finally {
-            try {
-                content.close();
-            } catch ( IOException e ) {
-                throw new org.uberfire.java.nio.IOException( e.getMessage() );
-            }
-        }
+        return create(resource, content, sessionId, comment, true);
     }
 
+    @Override
     public Path save( final Path resource,
                       final InputStream content,
                       final String sessionId,
                       final String comment ) {
+        return create(resource, content, sessionId, comment, false);
+    }
+
+    private Path create( final Path resource,
```

Comment:
Same as for the Decision Table service (e.g. ```writeToFile(...)```).. any reason the method is ```private``` here but package protected in ```DecisionTableXLSServiceImpl```?